website/pages/docs/concepts/seal.mdx

```diff
@@ -141,37 +141,48 @@ These steps are common for seal migrations between any supported kinds and for
 any storage backend.
 
 1. Take a standby node down and update the [seal
-   configuration](/docs/configuration/seal). If the migration is from Shamir seal
-   to KMS seal, add the desired new KMS seal block to the config. If the migration
-   is from KMS seal to Shamir seal, add `disabled = "true"` to the old seal block.
-   If the migration is from KMS seal to another KMS seal, add `disabled = "true"`
-   to the old seal block and add the desired new KMS seal block. Now, bring the
-   standby node back up and run the unseal command on each by supplying the
-   `-migrate` flag. Supply Shamir unseal keys if old seal was Shamir, which will be
-   migrated as the recovery keys for the KMS seal. Supply recovery keys if the old
-   seal is one of KMS seals, which will be migrated as the recovery keys of the new
-   KMS seal or as Shamir unseal keys if the new seal is Shamir.
-
-2. Perform step 1 for all the standby nodes, one at a time. It is necessary to
+   configuration](/docs/configuration/seal). 
+
+    * If the migration is from Shamir seal to KMS seal, add the desired new KMS
+      seal block to the configuration. 
+    * If the migration is from KMS seal to Shamir seal, add `disabled = "true"`
+      to the old seal block.
+    * If the migration is from KMS seal to another KMS seal, add `disabled =
+      "true"` to the old seal block and add the desired new KMS seal block. 
+
+    Now, bring the standby node back up and run the unseal command on each key, by
+    supplying the `-migrate` flag. 
+
+    * Supply Shamir unseal keys if old seal was Shamir, which will be migrated
+      as the recovery keys for the KMS seal. 
+    * Supply recovery keys if the old seal is one of KMS seals, which will be
+      migrated as the recovery keys of the new KMS seal, or as Shamir unseal
+      keys if the new seal is Shamir.
+
+1. Perform step 1 for all the standby nodes, one at a time. It is necessary to
```

Comment:
One thing @rberlind pointed out to me earlier while testing this, is that it might be helpful to add "expected" output for operators. Maybe a specific logline or subset of a logline.

Reply:
Can you tell me what you were thinking we could add here?  Maybe something about how to monitor the integrated storage status?

Reply:
Yeah that could be a good idea. Is there something that an operator should see regardless of the HA/storage engine used?